recipes/pandavro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pandavro" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd4ede1600cfc08150f28a7a1c1dd4abfbc87d76cd4a5b1639c8f55a04e97914
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
```

Comment:
conda-forge does not currently support python2 by default now. If you are fine with python3 only, I would replace this with `python >= 3.5` here and under `run`.  If you need python2.7, please see:

https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-python-2-7

Reply:
I'll leave the support of python >=3.5 only